pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -0,0 +1,1047 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for DatetimeIndex timezone-related methods
+"""
+from datetime import datetime, timedelta, tzinfo
+from distutils.version import LooseVersion
+
+import pytest
+import pytz
+import dateutil
+from dateutil.tz import gettz, tzlocal
+import numpy as np
+
+import pandas.util.testing as tm
+import pandas.util._test_decorators as td
+
+import pandas as pd
+from pandas._libs import tslib
+from pandas._libs.tslibs import timezones
+from pandas.compat import lrange, zip
+from pandas import (DatetimeIndex, date_range, bdate_range,
+                    Timestamp, isna, to_datetime, Index)
+
+
+class FixedOffset(tzinfo):
+    """Fixed offset in minutes east from UTC."""
+
+    def __init__(self, offset, name):
+        self.__offset = timedelta(minutes=offset)
+        self.__name = name
+
+    def utcoffset(self, dt):
+        return self.__offset
+
+    def tzname(self, dt):
+        return self.__name
+
+    def dst(self, dt):
+        return timedelta(0)
+
+
+fixed_off = FixedOffset(-420, '-07:00')
```

Comment:
make these into fixtures

Reply:
They're not being passed into parametrize anywhere.

Reply:
hmm, these should be then.